src/index.ts

```diff
@@ -506,9 +506,10 @@ export class SavParser {
             },
             display : partial.display ?? [],
             documents : partial.documents ?? [],
+            names : partial.names ?? new Map(),
```

Comment:
same here with `labels` being removable. Do not think they will be populated anymore.